app/src/main/java/com/infomaniak/mail/ui/main/folder/ThreadListAdapter.kt

```diff
@@ -218,6 +218,15 @@ class ThreadListAdapter @Inject constructor(
 
             draftPrefix.isVisible = hasDrafts
 
+            iconReply.isGone = true
+            iconForward.isGone = true
+
+            if (thread.isAnswered) {
+                iconReply.isVisible = true
+            } else if (thread.isForwarded) {
+                iconForward.isVisible = true
+            }
```

Comment:
Make sure how priority should truly be computed

Reply:
The priority is given to the last action on a message (replied or forwarded)